test_common.go

```diff
@@ -44,3 +118,91 @@ func mustHexDecode(t *testing.T, s string) []byte {
 	}
 	return data
 }
+
```

Comment:
It's at least the third time we are reusing this code in different packages.  We should start looking at how to factor this out :-)

Reply:
Agree. I have created https://github.com/veraison/psatoken/issues/39 to track this!